README.md

```diff
@@ -89,6 +89,7 @@ Pass these options to the HoneycombWebSDK:
 | spanProcessors | optional                                         | SpanProcessor[] | | Array of [span processors](https://opentelemetry.io/docs/languages/java/instrumentation/#span-processor) to apply to all generated spans.  |
 |  webVitalsInstrumentationConfig|optional|WebVitalsInstrumentationConfig| `{ enabled: true }` | See [WebVitalsInstrumentationConfig](####WebVitalsInstrumentationConfig). |
 |  globalErrorsInstrumentationConfig |optional| GlobalErrorsInstrumentationConfig|  `{ enabled: true }` | See [GlobalErrorsInstrumentationConfig](####GlobalErrorsInstrumentationConfig).
+| logLevel              | optional                                    | DiagLogLevel | DiagLogLevel.INFO       | Controls the verbosity of logs printed to the console. Currently, only controls logs printed for SDK option validation. |
```

Comment:
Looks like were saying the default is `INFO` here, but in the comment we're saying `Defaults to 'DEBUG'.`  which seems to be the case: `  const logLevel: DiagLogLevel = options?.logLevel
    ? options.logLevel
    : DiagLogLevel.DEBUG;`
    
```suggestion
| logLevel              | optional                                    | DiagLogLevel | DiagLogLevel.DEBUG       | Controls the verbosity of logs printed to the console. Currently, only controls logs printed for SDK option validation. |
```